tock/tock/settings/base.py

```diff
@@ -134,4 +134,5 @@
 SECURE_HSTS_INCLUDE_SUBDOMAINS = True
 SECURE_HSTS_PRELOAD = True
 
-RECENT_TOCKS_TO_REPORT = 5
\ No newline at end of file
+RECENT_TOCKS_TO_REPORT = 5
+STARTING_FY_FOR_REPORTS_PAGE = 2019
```

Comment:
Wondering if it would make sense to set the year dynamically from datetime.date. It wouldn't align with FY, but it would just overlap for few months and then update automatically. Not necessary, just thinking out loud.

Reply:
Good idea, that I think could extend to abstracting out the Fiscal Year math from `ReportingPeriod`.

Will open a new issue for that effort